scanner_test.go

```diff
@@ -9,18 +9,17 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"io"
-	"reflect"
-	"sync"
-	"testing"
-
 	"github.com/golang/mock/gomock"
 	"github.com/tsuna/gohbase/hrpc"
 	"github.com/tsuna/gohbase/pb"
 	"github.com/tsuna/gohbase/region"
 	"github.com/tsuna/gohbase/test"
 	"github.com/tsuna/gohbase/test/mock"
 	"google.golang.org/protobuf/proto"
+	"io"
```

Comment:
could you move these back up as once we enable linter, it will complain

Reply:
Hmm, they were moved by gofmt. If I move back, make fmt complains.

Reply:
that's weird, which version of go are you using? you also need to add a blank line between them

Reply:
1.14 

I can add a blank line.

Reply:
added